examples/cluster/plot_birch_vs_minibatchkmeans.py

```diff
@@ -18,6 +18,7 @@
 #          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
 # License: BSD 3 clause
 
+from os import cpu_count
```

Comment:
The `os` module of Python is not aware of docker container CPU usage constraints and using it might lead to report overly large number of CPUs while only a few of them are usable. I would therefore recommend to use joblib that can take those constraints into account.

```suggestion
from joblib import cpu_count
```